src/main/java/org/levelup/labradoodle/config/ApplicationConfig.java

```diff
@@ -28,6 +28,15 @@
     @Value("${LD_PORT:8080}")
     private int port;
 
+    @Value("${URL}")
+    private String url;
+
+    @Value("${USERNAME}")
+    private String username;
+
+    @Value("${PASSWORD}")
+    private String password;
+
```

Comment:
url, user, password... for what this values? Add prefixes, like {POSTGRES_URL}, ...